src/views/ApartmentApp/components/StudentApplication/components/ApplicationDataTable/index.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {
+  Card,
+  CardContent,
+  CardHeader,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableRow,
+  Typography,
+} from '@material-ui/core/';
+import EmailIcon from '@material-ui/icons/Email';
+import { DateTime } from 'luxon';
+
+/**
+ * Renders a card displaying a table of data about the current application
+ * @param {Object} props The React component props
+ * @param {DateTime} props.dateSubmitted The date the application was submitted
+ * @param {DateTime} props.dateModified The date the application was last modified
+ * @param {String} props.editorUsername The username of the application's editor
+ * @returns {JSX.Element} JSX Element for the data table card
+ */
+const ApplicationDataTable = ({ dateSubmitted, dateModified, editorUsername, editorEmail }) => {
+  function createData(label, value) {
+    return { label, value };
+  }
+
+  let rows = [
+    createData(
+      'Last Submitted: ',
+      DateTime.fromJSDate(dateSubmitted).toISODate() ?? 'Not yet submitted',
+    ),
+    createData('Last Modified: ', DateTime.fromJSDate(dateModified).toISODate() ?? 'Not yet saved'),
```

Comment:
`dateSubmitted` and `dateModified` are not JavaScript dates, they are strings, so this is always failing and returning `Not yet submitted`/`Not yet saved`. 

Instead, you can use Luxon to convert from a string directly:
```suggestion
      DateTime.fromISO(dateSubmitted).toISODate() ?? 'Not yet submitted',
    ),
    createData('Last Modified: ', DateTime.fromISO(dateModified).toISODate() ?? 'Not yet saved'),
```

Reply:
That makes sense, I was having a hard time figuring out exactly how the date information is sent from the backend to the frontend. **You may commit this suggestion for me**, since I do not actually have write access to this branch directly due to the protected-branch settings